docs/user/forms.md

```diff
@@ -95,7 +95,7 @@ However, while similar, there are some very important differences between the tw
 
 However, it is also important to note that the two lists do not *always* refer to the same underlying PDF object. For example, if the form contains radio buttons, you will find that `reader.get_fields()` will get the parent object (the group of radio buttons) whereas `page.annotations` will return all the child objects (the individual radio buttons).
 
-__Caution: Remember that fields are not stored in pages; if you use  `add_page()` the field structure is not copied. It is recommended to use `.append()` with the proper parameters instead.__
+__Remember that fields are not stored in pages; if you use  `add_page()` the field structure is not copied. It is recommended to use `.append()` with the proper parameters instead.__
```

Comment:
Could we convert this to a real warning box instead of just increasing the font weight?

Reply:
Done, but could you check?